client/CONTRIBUTING.md

```diff
@@ -1 +1,21 @@
 # Contributing to Batch SQL Translation Client
+
+## Install
+
+Install the client dependencies (including dev dependencies) in a 
+virtualenv.
+
+```shell
+git clone git@github.com:google/dwh-migration-tools.git
+cd dwh-migration-tools
+python3 -m venv venv
+source venv/bin/activate
```

Comment:
This is excellent; could well be wrapped into a shell script rather than a C&P. Perhaps this could even be wrapped into bin/dwh-migration-client so that it's a total no-brainer.

Reply:
Argh, I want to comment and approve, and I can't do that any more.